Gems/Atom/RHI/Vulkan/Code/Source/RHI/FrameGraphCompiler.h

```diff
@@ -100,6 +101,28 @@ namespace AZ
             VkPipelineStageFlags srcPipelineStageFlags = prevScopeAttachment ? GetResourcePipelineStateFlags(*prevScopeAttachment) : VK_PIPELINE_STAGE_BOTTOM_OF_PIPE_BIT;
             VkAccessFlags srcAccessFlags = prevScopeAttachment ? GetResourceAccessFlags(*prevScopeAttachment) : 0;
 
+            // this makes vkCmdClearColorImage is apart of this resource barrier
+            if constexpr (AZStd::is_same_v<ResourceScopeAttachment, RHI::ImageScopeAttachment>)
```

Comment:
Any reason why we wont need this for BufferScopeAttachment?

Reply:
Good point, let me double check how the buffer clear is done.